javascript/chronometer.js

```diff
@@ -1,37 +1,67 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(callback) {
-    // ... your code goes here
+    this.intervalId = setInterval(() => {
+      if (callback) {
+        callback();
+      }
+      this.currentTime++;
+    }, 1000);
+    return this.currentTime;
   }
 
   getMinutes() {
-    // ... your code goes here
+    let minutes = Math.floor(this.start() / 60);
+    return minutes;
   }
 
   getSeconds() {
-    // ... your code goes here
+    let seconds = this.start() % 60;
+    return seconds;
   }
 
   computeTwoDigitNumber(value) {
-    // ... your code goes here
+    let stringValue = (value).toString();
+    let paddedString;
+    if (stringValue.length === 0) {
+      return "00";
+    }
+    if (stringValue.length === 2) {
+      return stringValue;
+    } else if (stringValue.length === 1) {
+      paddedString = `0${stringValue}`;
+      return paddedString;
+    } else {
+      throw new Error("Something went wrong!");
+    }
```

Comment:
Basically a good solution and therefore you can leave it as it is. However, the solution is unnecessarily long and it would have been possible like this, for example:
```
computeTwoDigitNumber(number) {
    return ("0" + number).slice(-2);
  }
```
In this case I think that the long version is not necessarily more understandable

Reply:
nice solution - shorter and clearer, I'll remember this one